components/blitz/src/omero/gateway/model/TableDataColumn.java

```diff
@@ -0,0 +1,213 @@
+        this.index = index;
+        this.type = type;
+    }
+
+    /**
+     * Get the header
+     * 
+     * @return See above
+     */
+    public String getName() {
+        return name;
+    }
+
+    /**
+     * Set the header
+     * 
+     * @param name
+     *            The header
+     */
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    /**
+     * Get the description
+     * 
+     * @return See description
+     */
+    public String getDescription() {
+        return description;
+    }
+
+    /**
+     * Set the description
+     * 
+     * @param description
+     *            The description
+     */
+    public void setDescription(String description) {
+        this.description = description;
+    }
+
+    /**
+     * Get the index
+     * 
+     * @return See above
+     */
+    public int getIndex() {
+        return index;
+    }
+
+    /**
+     * Set the index
+     * 
+     * @param index
+     *            The index
+     */
+    public void setIndex(int index) {
+        this.index = index;
+    }
+
+    /**
+     * Get the data type
+     * 
+     * @return See above
+     */
+    public Class<?> getType() {
+        return type;
+    }
+
+    /**
+     * Set the data type
+     * 
+     * @param type
+     *            The data type
+     */
+    public void setType(Class<?> type) {
+        this.type = type;
+    }
+
+    @Override
+    public int hashCode() {
+        final int prime = 31;
+        int result = 1;
+        result = prime * result
+                + ((description == null) ? 0 : description.hashCode());
+        result = prime * result + index;
+        result = prime * result + ((name == null) ? 0 : name.hashCode());
+        result = prime * result
+                + ((type == null) ? 0 : type.getCanonicalName().hashCode());
+        return result;
+    }
+
+    @Override
+    public boolean equals(Object obj) {
+        if (this == obj)
+            return true;
+        if (obj == null)
+            return false;
+        if (getClass() != obj.getClass())
+            return false;
+        TableDataColumn other = (TableDataColumn) obj;
+        if (description == null) {
+            if (other.description != null)
+                return false;
+        } else if (!description.equals(other.description))
+            return false;
+        if (index != other.index)
+            return false;
+        if (name == null) {
+            if (other.name != null)
+                return false;
+        } else if (!name.equals(other.name))
+            return false;
+        if (type == null) {
+            if (other.type != null)
+                return false;
+        } else if (!type.getCanonicalName().equals(
+                other.type.getCanonicalName()))
+            return false;
+        return true;
+    }
+
+    @Override
+    public String toString() {
+        String s = name;
+        if (description.trim().length() > 0)
```

Comment:
``null`` check for description required
in that case, better to use method from  ``StringUtils``